### ISSUE TITLE
opcap fails on check - logs folder doesn't exist

### ISSUE THREAD

**Issue by acmenezes** [opdev/opcap#58]
@yashoza19 do we really need to store logs to files at this point? If so the file path is hard coded to a logs folder that is not created or existent on any system. I would advocate to avoid storing log files. Maybe let's use the root where opcap binary is seating no matter where it is. Users should have privileges to run and create files on that particular folder. Here is what happens:
```
panic: couldn't open sink "logs/stdout.json": open logs/stdout.json: no such file or directory

goroutine 1 [running]:
opcap/internal/logger.init.0()
        /Users/alex/go/src/github.com/acmenezes/opcap/internal/logger/zap.go:32 +0x1df
exit status 2
```

**Comment by yashoza19** [comment]
@acmenezes Thanks for pointing this out. I can change the hard coded path to stdout.json. This will create the logs file in the directory from which opcap binary is ran.

**Comment by acmenezes** [comment]
closed by #60
